### DISCUSSION TITLE
DBeaver not showing any connection drivers

### DISCUSSION THREAD

**Discussion by guilhermepereira2917** [dbeaver/dbeaver#21410] [DBeaver Development]
Hello, I would like to start contributing to DBeaver, I managed to build and run the repository on Eclipse following the installation instructions, but when I open DBeaver there are no database drivers available for me to connect to my test databases, I also can't install any additional drivers for some reason, as the add button is disabled for me. Can someone explain to me why is this happening and how would I solve this? I have the official DBeaver distribution installed and it works just fine, here are some screenshots of what is happening:

![image](https://github.com/dbeaver/dbeaver/assets/57549218/bbe803eb-c571-48da-9c1e-807cc2650d9f)

![image](https://github.com/dbeaver/dbeaver/assets/57549218/56c8accc-9e20-4ab3-b839-732ef01b329e)

As you can see in the console it loaded zero database drivers:

`
> Start Equinox Java Authentication and Authorization Service (JAAS) [org.eclipse.equinox.security 1.3.1000.v20220801-1135]
> Start DBeaver Application Standalone [org.jkiss.dbeaver.ui.app.standalone 23.2.1.qualifier]
> Start Eclipse IDE UI Application [org.eclipse.ui.ide.application 1.5.0.v20230526-1051]
> Start Expression Language [org.eclipse.core.expressions 3.9.0.v20230317-0901]
> Start DBeaver Desktop Application Core [org.jkiss.dbeaver.core 23.2.1.qualifier]
2023-10-02 14:41:11.561 - DBeaver 23.2.1.qualifier is starting
2023-10-02 14:41:11.562 - OS: Windows 10 10.0 (amd64)
2023-10-02 14:41:11.563 - Java version: 17.0.8.1 by Eclipse Adoptium (64bit)
2023-10-02 14:41:11.563 - Install path: 'C:\eclipsercp'
2023-10-02 14:41:11.563 - Instance path: 'file:/C:/eclipsercp/runtime-DBeaver.product/'
2023-10-02 14:41:11.563 - Memory available 64Mb/1024Mb
2023-10-02 14:41:11.605 - Create display
> Start Eclipse IDE UI [org.eclipse.ui.ide 3.21.0.v20230526-1500]
2023-10-02 14:41:11.801 - Initializing DBeaver 23.2.1
2023-10-02 14:41:11.809 - Host plugin: org.jkiss.dbeaver.ui.app.standalone 23.2.1.qualifier
2023-10-02 14:41:11.809 - Initialize desktop platform...
2023-10-02 14:41:11.835 - Language for locale 'pt_BR' not found. Use default.
2023-10-02 14:41:11.953 - BounceCastle bundle found. Use JCE provider BC
> Start Core File Systems [org.eclipse.core.filesystem 1.10.0.v20230425-0513]
> Start Core Resource Management [org.eclipse.core.resources 3.19.0.v20230510-1240]
> Start Team Support Core [org.eclipse.team.core 3.10.0.v20230317-0901]
> Start DBeaver UI Navigator [org.jkiss.dbeaver.ui.navigator 1.0.115.qualifier]
2023-10-02 14:41:12.767 - Initialize base platform...
2023-10-02 14:41:12.969 - Platform initialized (1160ms)
> Start DBeaver UI Editors - Base [org.jkiss.dbeaver.ui.editors.base 1.0.115.qualifier]
2023-10-02 14:41:13.070 - Starting instance server at http://localhost:64123
2023-10-02 14:41:13.090 - Run workbench
> Start DBeaver UI [org.jkiss.dbeaver.ui 5.1.130.qualifier]
> Start Eclipse e4 Workbench SWT [org.eclipse.e4.ui.workbench.swt 0.17.0.v20230501-1823]
> Start Eclipse Application Services [org.eclipse.e4.core.services 2.4.0.v20230429-1914]
> Start Eclipse e4 core commands [org.eclipse.e4.core.commands 1.1.0.v20230501-1823]
> Start Eclipse Bindings Support [org.eclipse.e4.ui.bindings 0.14.0.v20230501-1823]
> Start Eclipse UI Application Services [org.eclipse.e4.ui.services 1.6.0.v20230501-1823]
> Start EMF XML/XMI Persistence [org.eclipse.emf.ecore.xmi 2.35.0.v20230801-1141]
> Start Eclipse e4 Workbench Add-ons [org.eclipse.e4.ui.workbench.addons.swt 1.5.0.v20230501-1823]
> Start Eclipse CSS SWT Theme Support [org.eclipse.e4.ui.css.swt.theme 0.14.0.v20230501-1533]
> Start Eclipse e4 Workbench SWT Renderer [org.eclipse.e4.ui.workbench.renderers.swt 0.16.0.v20230525-0648]
!SESSION 2023-10-02 14:41:09.667 -----------------------------------------------
eclipse.buildId=unknown
java.version=17.0.8.1
java.vendor=Eclipse Adoptium
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=pt_BR
Framework arguments:  -product org.jkiss.dbeaver.ui.app.standalone.product -eclipse.keyring C:\Users\Systempro\AppData\Roaming\DBeaverData\secure\secure_storage
Command-line arguments:  -product org.jkiss.dbeaver.ui.app.standalone.product -data C:\eclipsercp\workspace/../runtime-DBeaver.product -dev file:C:/eclipsercp/workspace/.metadata/.plugins/org.eclipse.pde.core/DBeaver.product/dev.properties -os win32 -ws win32 -arch x86_64 -consoleLog
!ENTRY org.eclipse.ui 2 0 2023-10-02 14:41:13.509
!MESSAGE Warnings while parsing the key bindings from the 'org.eclipse.ui.commands' and 'org.eclipse.ui.bindings' extension point
!SUBENTRY 1 org.eclipse.ui 2 0 2023-10-02 14:41:13.509
!MESSAGE Cannot bind to an undefined command: plug-in='org.jkiss.dbeaver.ui.app.standalone', id='org.eclipse.text.quicksearch.commands.quicksearchCommand'
> Start Eclipse Dependency Injection Extensions [org.eclipse.e4.core.di.extensions 0.18.0.v20230429-1914]
> Start Eclipse CSS SWT Support [org.eclipse.e4.ui.css.swt 0.15.0.v20230317-0802]
> Start jna [com.sun.jna 5.13.0]
2023-10-02 14:41:14.399 - Configure workbench window
2023-10-02 14:41:14.597 - Total database drivers: 0 (0)
> Start Help System Core [org.eclipse.help 3.10.0.v20230317-0802]
2023-10-02 14:41:14.723 - Create actions
2023-10-02 14:41:14.724 - Disable Eclipse action set 'org.eclipse.ui.WorkingSetActionSet'
2023-10-02 14:41:14.725 - Disable Eclipse action set 'org.eclipse.ui.actionSet.openFiles'
2023-10-02 14:41:14.725 - Disable Eclipse action set 'org.eclipse.ui.edit.text.actionSet.annotationNavigation'
Warning: NLS unused message: dialog_connection_edit_wizard_data_format in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_data_format_description in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_editors in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_editors_description in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_presentation in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_presentation_description in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_resultset in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_resultset_description in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_sql_code_completion in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_sql_code_completion_description in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_sql_editor in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_sql_editor_description in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_sql_formatting in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_sql_formatting_description in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_sql_processing in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: dialog_connection_edit_wizard_sql_processing_description in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: notification_org_jkiss_dbeaver_ui_actions_common_uuid_copy in: org.jkiss.dbeaver.core.CoreResources
Warning: NLS unused message: notification_org_jkiss_dbeaver_ui_actions_common_uuid_copy_text in: org.jkiss.dbeaver.core.CoreResources
2023-10-02 14:41:15.200 - Initialize workbench window
2023-10-02 14:41:15.207 - Finish initialization
Warning: NLS missing message: pref_page_content_editor_group_representation_content in: org.jkiss.dbeaver.ui.controls.resultset.internal.ResultSetMessages
> Start DBeaver UI Editors - Data [org.jkiss.dbeaver.ui.editors.data 1.0.115.qualifier]
> Start Internet Connection Management [org.eclipse.core.net 1.5.0.v20230317-0901]
!ENTRY org.eclipse.ui 2 2 2023-10-02 14:41:16.546
!MESSAGE Invalid preference category path: org.eclipse.debug.ui.ConsolePreferencePage (bundle: org.eclipse.ui.console, page: org.eclipse.ui.internal.console.ansi.preferences.AnsiConsolePreferencePage)
> Start DBeaver SQL Model [org.jkiss.dbeaver.model.sql 1.0.105.qualifier]
> Start DBeaver UI Editors - SQL [org.jkiss.dbeaver.ui.editors.sql 1.0.115.qualifier]
> Start GEF Classic Draw2d [org.eclipse.draw2d 3.13.0.202306071707]
> Start GEF Classic GEF (MVC) [org.eclipse.gef 3.14.0.202306071707]
> Start DBeaver ERD UI [org.jkiss.dbeaver.erd.ui 3.0.75.qualifier]
> Start DBeaver ERD Model [org.jkiss.dbeaver.erd.model 1.0.75.qualifier]
> Start Default Text Editor [org.eclipse.ui.editors 3.16.0.v20230329-1609]
> Start Text Editor Framework [org.eclipse.ui.workbench.texteditor 3.17.0.v20230316-1200]
> Start DBeaver UI Editors - Connections [org.jkiss.dbeaver.ui.editors.connection 1.0.106.qualifier]
2023-10-02 14:41:19.588 - Open [https://dbeaver.io/product/version.xml]

**Comment by LonwoLonwo** [comment]
Hello,

Could you please check that Debug Configurations - Plug-ins contains all neccesary plugings? Like PostgreSQL, Oracle etc. And that they are checked.

**Reply by guilhermepereira2917** [reply]
It worked! Thank you very much, I actually selected only the required plugins because I was getting the "No application id has been found" error,  as the installation says to do. But it's working now, thank you again.
